docs/manage-certificate.md

```diff
@@ -0,0 +1,65 @@
+
+# Managing SmartThings&trade; Certificates
+
```

Comment:
If you provide contents at start of doc, it makes easy to know and easy to find what and where it is.
```
## Contents
> [Create certi....](#create...)
> [Import xxx](#...)
> [Activate xxx](#...)
```